scripts/remove.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* eslint no-restricted-syntax: 0 */
+const del = require('del');
+const path = require('path');
+
+if (!process.argv.includes(__filename)) {
+  console.error('Usage: node scripts/remove.js <target ...>');
+  process.exit(1);
+}
+
+const toDeletes = process.argv
+  .slice(process.argv.indexOf(__filename + 1))
+  .map((item) => path.resolve(item));
+
+if (toDeletes.length === 0) {
+  console.warn('Nothing to delete');
+  process.exit(0);
+}
+
+(async () => {
+  const deletedPaths = await del(toDeletes);
+  if (deletedPaths === 9) {
```

Comment:
Do we still need this if clause if process exit at #line21? why do we need to check again?

Reply:
I changed the message.

`toDeletes` is the requested list and `deletedPaths` is the list of the ones that actually got deleted.